<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>KbWidget</class>
 <widget class="QWidget" name="KbWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>773</width>
    <height>539</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="5" column="1">
    <spacer name="horizontalSpacer_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>0</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Serial number:</string>
     </property>
    </widget>
   </item>
   <item row="9" column="2">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="8" column="0">
    <widget class="QLabel" name="label_7">
     <property name="text">
      <string>Animation color:</string>
     </property>
    </widget>
   </item>
   <item row="7" column="0">
    <widget class="QLabel" name="label_6">
     <property name="text">
      <string>Animation:</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_8">
     <property name="text">
      <string>Layout:</string>
     </property>
    </widget>
   </item>
   <item row="5" column="2">
    <widget class="QComboBox" name="framerateBox">
     <item>
      <property name="text">
       <string>60</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>30</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>15</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Base color:</string>
     </property>
    </widget>
   </item>
   <item row="6" column="2">
    <widget class="ColorButton" name="bgButton">
     <property name="text">
      <string> #000000</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>FPS:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>Brightness:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="2" colspan="2">
    <widget class="QLabel" name="serialLabel">
     <property name="text">
      <string>N/A</string>
     </property>
    </widget>
   </item>
   <item row="8" column="2">
    <widget class="ColorButton" name="fgButton">
     <property name="text">
      <string> #FFFFFF</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>0</width>
       <height>30</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="5" column="3">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>     Try adjusting this if you experience animation issues.</string>
     </property>
    </widget>
   </item>
   <item row="5" column="4">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="7" column="2">
    <widget class="QComboBox" name="animBox">
     <item>
      <property name="text">
       <string>None</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Wave</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Ripple</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="3" column="2">
    <widget class="QComboBox" name="layoutBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <item>
      <property name="text">
       <string>United States</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>United Kingdom</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="QComboBox" name="brightnessBox">
     <item>
      <property name="text">
       <string>100%</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>67%</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>33%</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Off</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="1" column="1">
    <spacer name="verticalSpacer_3">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>0</width>
       <height>30</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_9">
     <property name="text">
      <string>Firmware version:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QLabel" name="fwLabel">
     <property name="text">
      <string>N/A</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ColorButton</class>
   <extends>QPushButton</extends>
   <header location="global">colorbutton.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
